_schemaV2.graphql

```diff
@@ -10930,6 +10966,24 @@ type GeneMeta {
   description: String!
 }
 
+# Abstract interface shared by every kind of home view section
+interface GenericHomeViewSection {
```

Comment:
It would be cool to have a `weight` field on this type that can tell us how to position a section relative to its siblings inside of the home page (or other containers we come up with).

We could then use this `weight` to sort and paginate the sections returned by a `HomeViewSectionConnection`.

Reply:
👍 on having the server return sections in the desired order.

Regarding the scenario you were thinking about: I envisioned clients always requesting sections ordered by `weight`, so the scenario you described shouldn't happen. However, having queries for sections always depend on a weight would be super limiting for us, so I think your idea of doing it server-side is more interesting.